packages/editor/src/components/post-card-panel/index.js

```diff
@@ -27,55 +27,47 @@ import {
 } from '../../store/constants';
 import { unlock } from '../../lock-unlock';
 
-export default function PostCardPanel( { actions } ) {
+export default function PostCardPanel( { postType, postId, actions } ) {
```

Comment:
The main change in this component is that it no longer relies on the "editor" store context to be able to work. It's actually a component that probably don't belong in "editor" package anymore but more something for "core-data" or "core-dataviews" or something like that.

Reply:
Can you expand a bit on this? Not that it is important for this PR.

In my mind, core-data is about data fetching and dataviews is about building lists & forms and it's WordPress agnostic. A `PostCardPanel` (title summary for a post) smells to an utility that fits in an editor package: a package anyone would use to build editors. Isn't that the purpose of the editor package? Why would you think it no longer fits here?

Reply:
"Editor" is a package that is for rendering an "editor" for a given post type. Its main requirement is that most of its component should be wrapped by `EditorProvider` which initializes the store and everything.

On the other hand: Fields, Actions, PostCard component these are things that define "fields", and "actions" (the whole `editor/src/dataviews` folder) and a utility component for a given post type, post Id regardless of whether they are used in an editor or not. They are about defining "entities" properly so that's why in my mind they are closer to `core-data` than `editor`. I would be ok with a dedicated package as well on top of `core-data`